src/main/java/net/javadiscord/javabot2/Bot.java

```diff
@@ -46,6 +52,11 @@
 	 */
 	public static ScheduledExecutorService asyncPool;
 
+	/**
+	 * The message cache.
+	 */
+	public static ConcurrentHashMap<MessageAuthor, LinkedList<Message>> messageCache;
```

Comment:
Don't use a specific collection implementation for declaring variable types, just use Map, List, Set, etc.